cli/src/main/java/org/aya/cli/repl/jline/completer/AyaCompleters.java

```diff
@@ -49,14 +51,23 @@ public Context(@NotNull Repl repl) {
       this.repl = repl;
     }
 
+    private @NotNull Tuple2<String, Boolean> fixWord(@NotNull String word, ParsedLine line) {
+      if (word.startsWith(":") || word.startsWith(Constants.SCOPE_SEPARATOR)) {
+        var idx = line.wordIndex();
+        if (idx >= 1) return Tuple.of(line.words().get(idx - 1) + word, true);
+      }
+      return Tuple.of(word, false);
+    }
+
     @Override public void complete(LineReader reader, ParsedLine line, List<Candidate> candidates) {
       var word = line.word();
+      var fixed = fixWord(word, line);
       var context = repl.replCompiler.getContext();
       context.modules.view().forEach((mod, contents) -> {
         var modName = mod.joinToString(Constants.SCOPE_SEPARATOR) + Constants.SCOPE_SEPARATOR;
-        if (!modName.startsWith(word)) return;
+        if (!modName.startsWith(fixed._1)) return;
         contents.keysView()
-          .map(name -> modName + name)
+          .map(name -> fixed._2 ? Constants.SCOPE_SEPARATOR + name : modName + name)
```

Comment:
`( ? : ) + name` should be better, agree?

Reply:
Yes!